packages/react-server-dom-webpack/npm/client.node.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
+var n, w;
 if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./cjs/react-server-dom-webpack-client.node.production.js');
+  n = require('./cjs/react-server-dom-webpack-client.node.production.js');
+  w = require('./cjs/react-server-dom-webpack-client.node-webstreams.production.js');
 } else {
-  module.exports = require('./cjs/react-server-dom-webpack-client.node.development.js');
+  n = require('./cjs/react-server-dom-webpack-client.node.development.js');
+  w = require('./cjs/react-server-dom-webpack-client.node-webstreams.development.js');
 }
+
+exports.registerServerReference = function (r, i, e) {
+  return w.registerServerReference(n.registerServerReference(r, i, e), i, e);
+};
+exports.createServerReference = function (i, c, e, d, f) {
```

Comment:
Will this wrapper mess with source mapping? I remember you mentioned in the past that module indirection is ok, but wrapper is not, because React doesn't let you customize how many stack frames to pop.

Reply:
That only matters on the `server` entry point and that one isn't wrapped because they are not using module state but instead attaches meta data on the function. I was hoping to change that to be the shared state model as the client but if we do that we'll just also have to hoist that shared state into a separate internal bundle.

Reply:
Actually I'm wrong about this. It doesn't matter for `registerServerReference` but it does matter for `createServerReference`.